events/importer/harrastushaku.py

```diff
@@ -64,7 +65,10 @@ def setup(self):
         self.data_source, _ = DataSource.objects.get_or_create(id=self.name, defaults={'name': 'Harrastushaku'})
         self.tprek_data_source = DataSource.objects.get(id='tprek')
         self.ahjo_data_source, _ = DataSource.objects.get_or_create(id='ahjo', name='Ahjo')
-        self.organization = Organization.objects.get(origin_id='u48040030', data_source=self.ahjo_data_source)
+        try:
```

Comment:
I would just use `get_or_create` instead, the same way that the other importers do this.

This way, each importer can be run even without running the whole `import_organizations` thing, if the user doesn't want hierarchical organization structure quite yet.